### DISCUSSION TITLE
Source code of: "Remove Unicode control characters..."

### DISCUSSION THREAD

**Discussion by ghost** [SubtitleEdit/subtitleedit#5926] [General]
hello, i'm looking for the source code of the option: "Remove Unicode control characters (for selected lines)"
or if someone can explain me the exact characters it looks for? 

thanks :)

**Comment by OmrSi** [comment]
Here you go: https://github.com/SubtitleEdit/subtitleedit/blob/master/src/libse/Common/Utilities.cs#L2904
